src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxFacts.cs

```diff
@@ -1268,6 +1268,12 @@ public SeparatedSyntaxList<SyntaxNode> GetArgumentsOfObjectCreationExpression(Sy
         public SeparatedSyntaxList<SyntaxNode> GetArgumentsOfArgumentList(SyntaxNode? argumentList)
             => (argumentList as BaseArgumentListSyntax)?.Arguments ?? default;
 
+        public SyntaxNode GetArgumentListOfInvocationExpression(SyntaxNode invocationExpression)
+            => (invocationExpression as InvocationExpressionSyntax)!.ArgumentList;
+
+        public SyntaxNode? GetArgumentListOfObjectCreationExpression(SyntaxNode objectCreationExpression)
+            => (objectCreationExpression as ObjectCreationExpressionSyntax)!.ArgumentList;
```

Comment:
so these are inconsistent with the expected pattern.  teh `Get` helpers shoudl not do `as` casts.  (i agree we're not consistent with this) :)

Reply:
fixed